packages/web-app-files/src/composables/keyboardActions/useKeyboardActionsSearchTable.ts

```diff
@@ -1,11 +1,12 @@
-import { computed, onBeforeUnmount, onMounted, ref, unref } from 'vue'
+import { Ref, computed, nextTick, onBeforeUnmount, onMounted, ref, unref } from 'vue'
 import { useStore } from 'web-pkg/src/composables'
 import { useScrollTo } from 'web-app-files/src/composables/scrollTo'
 import { Key, ModifierKey } from 'web-pkg/src/composables/keyboardActions'
 import { eventBus } from 'web-pkg'
 import { useGettext } from 'vue3-gettext'
+import { Resource } from 'web-client/src'
 
-export const useKeyboardActionsSearchTable = (keyActions, paginatedResources) => {
+export const useKeyboardActionsSearchTable = (keyActions, paginatedResources: Ref<Resource[]>) => {
```

Comment:
@jacob-nv hope that little update is fine with you? :)

Reply:
Completely fine, this composable is being split up in new updates coming